### PR TITLE
Feature Food Details, Profile page, Food Categories

### PR DESCRIPTION
**Pull request by ArmantasPik**

### CODE REVIEW

File: Pages/FoodCustomer/Details.cshtml.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+            //Configure Lat Lng
+            if(Food.ApplicationUser.Lat != 0 && Food.ApplicationUser.Lng != 0)
+            {
+                Lat = Food.ApplicationUser.Lat;
+                Lng = Food.ApplicationUser.Lng;
+            }
+            else
+            {
+                Lat = 54.68587937724495;
```
Comment (medikornov) [inline Pages/FoodCustomer/Details.cshtml.cs:51]: ? what these numbers mean

**Review by 0rl1s — APPROVED** [review]

File: Pages/FoodCustomer/Details.cshtml.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    public class DetailsModel : PageModel
+    {
+        private readonly FoodKept.Data.ShopContext _context;
+
+        public DetailsModel(FoodKept.Data.ShopContext context)
+        {
+            _context = context;
+        }
+
+        public Food Food { get; set; }
```
Comment (medikornov) [inline Pages/FoodCustomer/Details.cshtml.cs:25]: why these are static

File: Pages/FoodCustomer/Details.cshtml.cs [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        public static double Lat { get; set; }
+        public static double Lng { get; set; }
+
```
Comment (medikornov) [inline Pages/FoodCustomer/Details.cshtml.cs:28]: why these are static

Comment (ArmantasPik) [reply Pages/FoodCustomer/Details.cshtml.cs:28]: Accessed in script so need to be static or an error is shown

**Pull request merged**